dowhy/causal_estimators/regression_estimator.py

```diff
@@ -111,7 +110,15 @@ def _do(self, treatment_val, data_df=None):
             # The model is always built on the entire data
```

Comment:
what's the expected type for `treatment_val`? Numpy array of 2 dimensions?

Reply:
It is meant to be consistent with what DoWhy was previously doing. So previously I think you were able to accept a scalar `1` or  a tuple `(1, 1)` for multiple treatments or a numpy array. I am using `np.full` as before, and categoricals here should be coded using integers (so just using one column per variable as before). The main change is that I call `get_dummies` later on in the function now